widget/embedded/src/components/ItemPicker/ItemPicker.tsx

```diff
@@ -0,0 +1,57 @@
+import type { PropTypes } from './ItemPicker.types';
+
+import { ChevronRightIcon, Divider, Image, Typography } from '@rango-dev/ui';
+import React from 'react';
+
+import { Container, FlexContainer, InputContainer } from './ItemPicker.styles';
+
+function ItemPicker(props: PropTypes) {
```

Comment:
`Item` is too general for this component, and also `picker` is suitable for cases you are picking a value from a list that passed to component.

Reply:
I suggest `BlockchainSelectorButton` or something like this.

Reply:
fixed